images/stackstorm-all/Dockerfile

```diff
@@ -0,0 +1,56 @@
+####################################
+# Build the image:
+#
+#   docker build -t stackstorm/st2_aio .
+#	You need to be in the dirctory that contains the Dockerfile.
```

Comment:
I think these general comments that are not directly related to the content of Dockerfile can be moved and merged into `README.md`